src/domain/Users/services/listUsers.service.test.ts

```diff
@@ -0,0 +1,13 @@
+import { container } from "tsyringe";
+import { USERS_LIST_MOCK } from "../repositories/repositoryMock";
+import { type ListUsersService } from "./listUsers.service";
+
+describe("List of Users", () => {
```

Comment:
Could you replace this as well to follow a pattern?
```suggestion
describe("service: ListUsersService", () => {
```